cmd/restic/main.go

```diff
@@ -104,7 +104,11 @@ func main() {
 
 	var exitCode int
 	if err != nil {
-		exitCode = 1
+		if errors.Is(err, errors.InvalidSourceData) {
+			exitCode = 2
```

Comment:
Go already uses exit code 2 when crashing due to an uncaught panic: https://github.com/golang/go/commit/f947c4dcfe6f22e6c92e6e5a170096b518a18448

Reply:
Are there any other exit codes go uses or is 3 okay?

Reply:
Are there any other exit codes that go uses or is using 3 okay?

Reply:
Exit code 3 seems to be fine. http://www.tldp.org/LDP/abs/html/exitcodes.html only mentions 1 and 2 as used.